src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -216,6 +216,7 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.ConfigureOptions<FlexConsumptionMetricsPublisherOptionsSetup>();
             services.ConfigureOptions<ConsoleLoggingOptionsSetup>();
             services.AddHostingConfigOptions(configuration);
+            services.ConfigureOptions<ExtensionRequirementOptionsSetup>();
```

Comment:
Do we need a change token like we do when we configure `FunctionsHostingConfigOptions>` via `AddHostingConfigOptions` ?

Reply:
We don't need that as we don't expect this file to change.